docs/reference/content/bson/extended-json.md

```diff
@@ -116,5 +116,29 @@ try {
 The `Document.parse()` static factory method constructs an instance of a `JsonReader` with the given string and returns an instance of an
 equivalent Document. `JsonReader` automatically detects the JSON flavor in the string, so you do not need to specify it.
 
+### Reading Directly to JSON
+If you do not need a document and only want to deal with JSON, you can use the `JsonStringCodec` to read JSON directly from the database. 
+This is more efficient than constructing a `Document` first and then calling `toJson()`. `JsonStringCodec` is part of the default registry, 
```

Comment:
Also more efficient than calling Document#parse, so point that out as well.

Reply:
Done